airmozilla/manage/static/manage/js/picturegallery.js

```diff
@@ -240,6 +248,26 @@ app.controller('PictureGalleryController', ['$scope', '$http',
             });
         };
 
+        $scope.deleteAllPictures = function() {
+            var csrf = $('input[name="csrfmiddlewaretoken"]').val();
+            var url = $scope.url('manage:picture_delete_all');
+            var data = {
+                'csrfmiddlewaretoken': csrf,
+                'event': $scope.current_event
+            };
+            $http({
+                method: 'POST',
+                url: url,
+                headers: {'Content-Type': 'application/x-www-form-urlencoded'},
+                data: serializeObject(data)
+            }).success(function(data) {
+                $scope.pictures = [];
```

Comment:
I don't think $scope.pictures = [] is the right approach because if there is a picture in this view which does not have an event_id (just some random picture) then it would appear as if it's been deleted but if you refresh the page , that of course would not be the case. I think maybe we should pass in the picture.event_id info to the context and only remove those pictures from the view that have a picture.event_id.